src/legacy/status_im/utils/test.cljs

```diff
@@ -31,108 +31,100 @@
                       (.pollSignal native-status signal-received-callback)
                       100))))
 
-(def status
+(def ui-helper
   (clj->js
-   {:openAccounts
-    (fn [callback]
-      (callback (.openAccounts native-status test-dir)))
-
-    :multiAccountStoreDerived
-    (fn [json callback]
-      (callback (.multiAccountStoreDerivedAccounts native-status json)))
-
-    :getNodeConfig (fn [] (types/clj->json {:WakuV2Config ""}))
+   {:clearCookies     identity
+    :clearStorageAPIs identity}))
 
-    :backupDisabledDataDir (fn [] (str test-dir "/backup"))
-    :keystoreDir (fn [] "")
-    :logFileDirectory (fn [] (str test-dir "/log"))
-    :clearCookies identity
-    :clearStorageAPIs identity
+(def encryption-utils
+  (clj->js
+   {:sha3
+    (fn [s] (.sha3 native-status s))
     :setBlankPreviewFlag identity
+    :encodeTransfer
+    (fn [to-norm amount-hex]
+      (.encodeTransfer native-status to-norm amount-hex))
+    :encodeFunctionCall
+    (fn [method params-json]
+      (.encodeFunctionCall native-status method params-json))
+    :hexToNumber
+    (fn [hex] (.hexToNumber native-status hex))
+    :decodeParameters
+    (fn [decode-param-json]
+      (.decodeParameters native-status decode-param-json))
+    :numberToHex
+    (fn [num-str] (.numberToHex native-status num-str))
+    :initKeystore
+    (fn [key-uid callback]
+      (callback (.initKeystore native-status
+                               (str test-dir "/keystore/" key-uid))))
+    :multiformatDeserializePublicKey
+    (fn [public-key deserialization-key callback]
+      (callback (.multiformatDeserializePublicKey
+                 native-status
+                 public-key
+                 deserialization-key)))}))
 
-    :callPrivateRPC
-    (fn [payload callback]
-      (callback (.callPrivateRPC native-status payload)))
-
-    :createAccountAndLogin (fn [request] (.createAccountAndLogin native-status request))
-
-    :saveAccountAndLogin
+(def account-manager
```

Comment:
I don't know about you and other folks, but the lack of consistent whitespace in the vars `account-manager`, `utils`, and `encryption-utils` makes reading them uncomfortable.

My feedback about the code style is to separate different keys and values by a line break, and to consistently move the function body below the arguments' vector in each `(fn [...] <function body>)`. This makes scanning the code vertically a breeze.

```clojure
(def account-manager
  (clj->js
   {:saveAccountAndLogin
    (fn [multiaccount-data password settings config accounts-data]
      (.saveAccountAndLogin native-status multiaccount-data password settings config accounts-data))

    :openAccounts
    (fn [callback]
      (callback (.openAccounts native-status test-dir)))

    :createAccountAndLogin
    (fn [request]
      (.createAccountAndLogin native-status request))

    :logout
    (fn []
      (.logout native-status))

    :multiAccountImportMnemonic
    (fn [json callback]
      (callback (.multiAccountImportMnemonic native-status json)))

    :multiAccountLoadAccount
    (fn [json callback]
      (callback (.multiAccountLoadAccount native-status json)))

    :multiAccountDeriveAddresses
    (fn [json callback]
      (callback (.multiAccountDeriveAddresses native-status json)))

    :multiAccountGenerateAndDeriveAddresses
    (fn [json callback]
      (callback (.multiAccountGenerateAndDeriveAddresses native-status json)))

    :multiAccountStoreDerived
    (fn [json callback]
      (callback (.multiAccountStoreDerivedAccounts native-status json)))}))
```

Reply:
Ah yes thanks for this suggestion, I did keep a mental note back when I was fixing tests that this formatting did look weird.
I later forgot about it and assumed that `make lint-fix` would tidy things up 😄 
I'll tidy this up.

Reply:
addressed here : https://github.com/status-im/status-mobile/pull/18385/commits/c877bd3c36c7caa0e99757c9715148cc7e51fa73